cli/main.go

```diff
@@ -147,6 +135,7 @@ func main() {
 				Aliases: []string{"cp"},
 				Usage:   "Generates a proof for use with EigenPod.verifyCheckpointProofs().",
 				Flags: []cli.Flag{
+					BatchBySize(&batchSize, 80),
```

Comment:
Doesn't use the constants defined in `checkpoint.go` and `validator.go`

Reply:
oh i guess there's only a const defined in `checkpoint.go`

if we're using this `BatchBySize` instead, let's define both the consts in main rather than using magic numbers here.

Reply:
Yeah agreed i'll fix